packages/model-typings/src/models/ILivechatRoomsModel.ts

```diff
@@ -234,4 +234,9 @@ export interface ILivechatRoomsModel extends IBaseModel<IOmnichannelRoom> {
 	setVisitorInactivityInSecondsById(roomId: string, visitorInactivity: any): Promise<UpdateResult>;
 	changeVisitorByRoomId(roomId: string, visitor: { _id: string; username: string; token: string }): Promise<UpdateResult>;
 	unarchiveOneById(roomId: string): Promise<UpdateResult>;
+	getConversationsBySource(start: Date, end: Date): AggregationCursor<{ data: { label: string; value: number }[] }>;
```

Comment:
Wouldn't be better to create a type for the returned data instead of repeating it?